.circleci/config.yml

```diff
@@ -5645,7 +5645,7 @@ workflows:
           context: org-member
           requires:
             - binary_windows_libtorch_3_7_cu102_release_nightly_test
-      - update_s3_htmls:
+      - update_s3_htmls_for_nightlies:
           context: org-member
           filters:
             branches:
```

Comment:
@seemethere It seems that we will need to split the filters.

Reply:
Alternatively, we can insert a step in update_s3_htmls to set `PIP_UPLOAD_FOLDER`.

Reply:
I switched to the second way as it sounds more reasonable.